docs/python/run.md

```diff
@@ -0,0 +1,56 @@
+
+### Native REPL
+
+The VS Code Native REPL for Python builds upon the classic Python REPL and provides additional features such as Intellisense and syntax highlighting to make your Python development experience more efficient. However, this REPL still adheres to principles present in the REPL built-in to Python itself, in that historical execution order and its content are immutable.
+
+You can open the Native REPL via the Command Palette (`kb(workbench.action.showCommands)`) by searching for **Python: Start Native REPL**. Furthermore, you can send code to the Native REPL via Smart Send (`kbstyle(Shift+Enter)`) and **Run Selection/Line** by setting `"python.REPL.sendToNativeREPL": true` in your `settings.json` file. You can opt to continue to use the REPL built-in to Python located in the terminal ( `>>>` ) by setting `"python.REPL.sendToNativeREPL": false` in your `settings.json`.
+
+![Gif showing the Native REPL for Python.](/docs/python/images/shared/native-repl-demo.gif)
+
+### Terminal REPL
+
+Similar to how you can interact with the Python REPL outside of VS Code, you can open a Terminal within VS Code and activate a Python REPL. To do so, you can search in the Command Palette (`kb(workbench.action.showCommands)`) for **Python: Start Terminal REPL** which opens a terminal for the currently selected Python interpreter. Alternatively, you can navigate to **Terminal > New Terminal** and type `python` (Windows) or `python3` (macOS/Linux) command.
+
+## Run code within your Python project
+
+The Python extension allows you to run Python code without any configuration needed.
+
+1. Click the **Run Python File in Terminal** play button in the top-right side of the editor.
+
+![Using the Run Python File in Terminal button](images/tutorial/run-python-file-in-terminal-button.png)
+
+The button opens a terminal panel in which your Python interpreter is automatically activated, then runs `python3 hello.py` (macOS/Linux) or `python hello.py` (Windows):
+
+![Program output in a Python terminal](images/tutorial/output-in-terminal.png)
+
+2. Right-click anywhere in the editor window and select **Run > Python File in Terminal** (which saves the file automatically):
+
+   ![Run Python File in Terminal command in the Python editor](images/tutorial/run-python-file-in-terminal.png)
+
+3. Select one or more lines, then press `kbstyle(Shift+Enter)` or right-click and select **Run Selection/Line in Python Terminal**. This command is convenient for testing just a part of a file.
+
+4. Place your cursor on a line of code and press `kbstyle(Shift+Enter)` to activate Smart Send.
+
+### Smart Send
```

Comment:
Thanks for adding this section for smart send - many folks wanted additional space that explains smart send.
For the beginners, it would also be really helpful to add two small video that runs Python code (say a dictionary) with & without smart send. This is in case they have trouble understanding how we describe smart send.